src/runner.js

```diff
@@ -0,0 +1,47 @@
+const {
+  getPackageInfoList, writeReportFile, extractInvalidPackages
+} = require('./utils');
+
+const run = async (checker, args) => {
+  const packages = await checker.parsePackages(args.start);
```

Comment:
❓  Why do we need to send the checker as an input argument instead of importing it here?

Reply:
I think it's easier for testing. We can use a mock to return whatever value we want, including an `error`

Reply:
🆗